api/src/routes/games/translations/Translations.ts

```diff
@@ -0,0 +1,184 @@
+import { Router } from 'express';
+import { prisma } from '../../../database/Database';
+import { Prisma } from '@prisma/client';
+
+const translationsRouter = Router();
+
+translationsRouter.get('/:slug/translations', async (req, res) => {
+    const { slug } = req.params;
+
+    const result = (await prisma.game.findUnique({ where: { slug } }))
+        ?.translations;
+    res.json(result);
+    return;
+});
+
+translationsRouter.post('/:slug/translations', async (req, res) => {
+    const { slug } = req.params;
+    const newTranslations = req.body.translations;
+    if (!Array.isArray(newTranslations)) {
+        res.status(400).json({
+            error: 'Translations must be an array',
+        });
+    }
+    const oldTranslations = (await prisma.game.findUnique({ where: { slug } }))
```

Comment:
Most (if not all) of this logic should be moved to one or more functions in a file in the database folder, Especially in this case where the function would be more than a simple wrapper around the prisma api. It makes the behavior more testable (when we eventually write the missing tests) by separating API responses from actual logic, and gives us a simple function we can easily call if we ever need to update translations from elsewhere (i.e. a different API endpoint) in the future).